spec/io_handler_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../io_handler'
+
+describe 'IOHandler' do
+  
+  mock_io_handler = MockIOHandler.new
+  prompter = Prompter.new
+  
+  describe 'printer' do
+
+    it 'will print game_greeting to terminal' do
+      mock_io_handler.printer(prompter.game_greeting)
+      expect(mock_io_handler.output_called_with).to eq "Welcome to Hangman!"
+    end
+
+  end
+
+  describe 'get_input' do
```

Comment:
This is the happy path when it works. Are you sufficiently accounting for bad input somewhere else in your tests?